dashboard/app/views/shared/_sharing_meta_tags.html.haml

```diff
@@ -0,0 +1,39 @@
+-# only works for pages that have a @level
+- if @level && @level.game.supports_sharing?
+  - content_for :head do
+    -# facebook boilerplate
+    = tag('meta', :property => 'fb:app_id', :content => '500177453358606')
```

Comment:
Nit: prefer new 1.9 symbol hash syntax: `content: '500177453358606'` but I realized this is old code.